### DISCUSSION TITLE
API Pass for HA

### DISCUSSION THREAD

**Discussion by AseKarlsson** [zachowj/hass-node-red#70] [Q&A] [answered]
Hi,

I run HA using docker on a Synology (1815+) and have couple of questions:
I downloaded nodered/node-red version 1.3.5 and installed in a different docker instance on the unit.

1.	Is this the right version to use on this Synology there was a lot of different versions such as 1.3.5-10, 1.3.5-12, 1.3.5-10-amd64 etc. What are the differences?
2.	I now have Node-Red up and running but it cannot talk to HA. I just get the error “Cannot GET /homeassistant/entities” when trying to do something. When trying to setup the server I have a field called “API Pass” how should this be setup in the HA end, I cannot find the config part that some guides talks about?
3.    After downloading the "Node-RED Companion" using HACS I see the text "To help integrate with Home Assistant Core", does Node-Red only work with the Core version of HA?

Thanks,
AseKarlsson

**Comment by zachowj** [comment, marked as answer]
> 1. Is this the right version to use on this Synology there was a lot of different versions such as 1.3.5-10, 1.3.5-12, 1.3.5-10-amd64 etc. What are the differences?

You would be better to ask on the NR forums. https://discourse.nodered.org/

>  I now have Node-Red up and running but it cannot talk to HA. I just get the error “Cannot GET /homeassistant/entities” when trying to do something. When trying to setup the server I have a field called “API Pass” how should this be setup in the HA end, I cannot find the config part that some guides talks about?

https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/

>  After downloading the "Node-RED Companion" using HACS I see the text "To help integrate with Home Assistant Core", does Node-Red only work with the Core version of HA?

No, make sure you have completed all the steps on installation.

**Reply by AseKarlsson** [reply]
Hi Jason,

Thanks for the link, working now!

//AseKarlsson
